master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -1,9 +1,14 @@
 ---
 title: Kubeadm Hosted Install
+redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/index
```

Comment:
We don't want to add this line to the master docs version.  Please remove it, thanks.